config-default.yml

```
@@ -231,11 +232,12 @@ guild:
         - *HELPERS_ROLE
 
     webhooks:
-        talent_pool:    569145364800602132
-        big_brother:    569133704568373283
-        reddit:         635408384794951680
-        duck_pond:      637821475327311927
-        dev_log:        680501655111729222
+        talent_pool:                  569145364800602132
+        big_brother:                  569133704568373283
+        reddit:                       635408384794951680
+        duck_pond:                    637821475327311927
+        dev_log:                      680501655111729222
+        python_news: &PYNEWS_WEBHOOK 704381182279942324
```

The ID of the webhook is not aligned with the other IDs in this section; it's missing one space character.

Also, for the other aliases/IDs, we typically align them on a tab stop (with the first character of the alias/ID starting at a column that's a multiple of 4). For these constants, this would mean this:

```suggestion
        talent_pool:                    569145364800602132
        big_brother:                    569133704568373283
        reddit:                         635408384794951680
        duck_pond:                      637821475327311927
        dev_log:                        680501655111729222
        python_news:    &PYNEWS_WEBHOOK 704381182279942324
```